### ISSUE TITLE
Unified Api support

### ISSUE THREAD

**Issue by welshrat** [cdeutsch/JsBridge#7]
Just wondered if this will support the new unified api within xamarin ?

**Comment by cdeutsch** [comment]
I haven't contributed code to this library for over a year, and I don't have plans to do so.

Would take a PR for adding support.

**Comment by welshrat** [comment]
Ok fair enough, will take a look at doing so myself if that is ok, seems to be just the NSUrlProtocolClient that is complaining

**Comment by cdeutsch** [comment]
Sounds good. 

I'll also ask around at work if anyone wants to take this over or if I can work on it during work hours. 

I'd love to see a V2 of this component with Android support but I'm mostly in the web world these days.

**Comment by welshrat** [comment]
Hey Christopher, I just wondered if you had got anywhere with this. I have been using my newly compiled version with the three lines referencing Client commented out and so far no issues have arisen, however I guess this isn't the best solution :-)

**Comment by cdeutsch** [comment]
Yes, I'm just waiting for version 8.6 of Xamarin.iOS to land in the Alpha channel.

**Comment by welshrat** [comment]
Excellent News, I shall bother you no longer. Many thanks for getting back to me
